src/core/pathfind.js

```diff
@@ -74,7 +78,7 @@ PathFind.prototype.notify_update = function (message) {
   }
 };
 
-PathFind.prototype.notify_superceded = function () {
+PathFind.prototype.notify_superceded = function() {
```

Comment:
I only changed calls that were broken by removing the attachUnderscored converter. Changing all underscore functions would probably be a bigger project.